smart_canvas/upload.py

```diff
@@ -0,0 +1,20 @@
+""" upload.py """
+import os
+import requests
+
+
+def upload_image(image_path, url=None, auth_token=None):
+    """
+    Uploads image to web-service
+    """
+    with open(image_path, 'rb') as img:
```

Comment:
Should we input cv2 type image instead?

Reply:
unless you need to see it before downloading, I don't think it matters